spec/system/admin/enterprises_spec.rb

```diff
@@ -725,6 +725,53 @@
             .to eq('Enterprise "First Distributor" has been successfully updated!')
           expect(distributor1.reload.hide_groups_tab).to be false
         end
+
+        context "creating custom tabs" do
+          before do
+            visit edit_admin_enterprise_path(distributor1)
+            within(".side_menu") do
+              click_link "White Label"
+            end
+            check "Create custom tab in shopfront"
+          end
+
+          it "display a form with the custom tab fields: title and content" do
+            expect(page).to have_content "TITLE FOR CUSTOM TAB"
+            expect(page).to have_field "enterprise_custom_tab_attributes_title"
+            expect(page).to have_content "CONTENT FOR CUSTOM TAB"
+            expect(page).to have_field "enterprise_custom_tab_attributes_content"
+          end
+
+          it "can save custom tab fields" do
+            fill_in "enterprise_custom_tab_attributes_title", with: "Custom tab title"
+            fill_in "enterprise_custom_tab_attributes_content", with: "Custom tab content"
+            click_button 'Update'
+            expect(flash_message)
+              .to eq('Enterprise "First Distributor" has been successfully updated!')
+            expect(distributor1.reload.custom_tab.title).to eq("Custom tab title")
+            expect(distributor1.reload.custom_tab.content).to eq("Custom tab content")
+          end
```

Comment:
If the labels are properly connected to the inputs, you should be able to use them with `fill_in`. In that case, you don't need a separate test for "display..." and filling in the fields using their label will proof the same thing. The purpose is editing here. Just seeing it is not much value for the user.

Reply:
Right, this test is a non sense: I created it in a TDD mode. 
That's why it only contains:

```
          it "display a form with the custom tab fields: title and content" do
            expect(page).to have_content "TITLE FOR CUSTOM TAB"
            expect(page).to have_content "CONTENT FOR CUSTOM TAB"
          end
```
Not that useful for sure. Would you like me to remove it?

Reply:
Yes, I'll quickly do it.